app/models/ems_azure.rb

```diff
@@ -0,0 +1,49 @@
+
+class EmsAzure < EmsCloud
```

Comment:
@bzwei 
That was discussed, and in the interest of time we decided to postpone until an upcoming sprint.